### DISCUSSION TITLE
Mock data with MSW

### DISCUSSION THREAD

**Discussion by thelafarge** [kubb-labs/kubb#1474] [General]
Hello,

This simple response generate `faker.string.alpha()`

```yaml
responses:
  '200':
    description: successful operation
    content:
      application/json:
        schema:
          type: string
          example: 'East Jarretmouth'
```

Do you know what can I do in my OpenAPI schema si kubb generate something more interesting than `faker.string.alpha()` ?

Also when I try to generate fake data for the MSW plugin with the parser set to "data" instead of "faker" no data is returned. I assume that it would have picked the example value ?

What am I missing ?

What is your workflow when using kubb ? Do you set `output.clean` to false in the config and edit manually the generated file to return what you want ?

**Comment by stijnvanhulle** [comment]
@thelafarge You can use `format`, things like uuid, email, uri will convert to its linked faker helper(same for Zod).

For creating more custom types, could you use `mapper`, see https://github.com/kubb-labs/kubb/blob/main/examples/zod/kubb.config.js#L40-L63
